### PR TITLE
Add loans table for mango, kamino

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

File: src/loans/kamino.py [hunk 1]
```diff
@@ -257,4 +257,112 @@ def compute_liquidable_debt_at_price(
...
+            / (10**decimals)
+            * token_prices[underlying_token]
+        )
+        
+        loan_states[f'risk_adj_collateral_usd_{token}'] = (
+            loan_states[f'collateral_{token}'].astype(float)
+            / (10**decimals)
+            * (ltv/100)
+            * token_prices[underlying_token]
+        )
```
Comment (lukaspetrasek) [inline src/loans/kamino.py:326]:
```suggestion
        loan_states[f'risk_adj_collateral_usd_{token}'] = loan_states[f'collateral_usd_{token}'] * (ltv/100)
```

File: src/loans/kamino.py [hunk 2]
```diff
@@ -257,4 +257,112 @@ def compute_liquidable_debt_at_price(
...
+        decimals = debt_token_parameters[debt_token]['underlying_decs']
+        ltv = debt_token_parameters[debt_token]['ltv']
+        underlying_token = src.kamino_vault_map.supply_vault_to_mint_map[debt_token]
+        loan_states[f'debt_usd_{debt_token}'] = (
+            loan_states[f'debt_{debt_token}'].astype(float)
+            / (10**decimals)
+            * token_prices[underlying_token]
+        )
+
+        loan_states[f'risk_adj_debt_usd_{debt_token}'] = (
```
Comment (lukaspetrasek) [inline src/loans/kamino.py:342]: DITTO

File: src/loans/kamino.py [hunk 3]
```diff
@@ -257,4 +257,112 @@ def compute_liquidable_debt_at_price(
...
+            * (1/(ltv/100) if ltv else 1)
+            * token_prices[underlying_token]
+        )
+    loan_states['risk_adj_debt_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('risk_adj_debt_usd_')]].sum(axis=1)
+    loan_states['debt_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('debt_usd_')]].sum(axis=1)
+
+    loan_states['risk_adj_collateral_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('risk_adj_collateral_usd_')]].sum(axis=1)
+    loan_states['collateral_usd'] = loan_states[[i for i in loan_states.columns if i.startswith('collateral_usd_')]].sum(axis=1)
+
+    loan_states['std_health'] = (loan_states['risk_adj_collateral_usd'] / loan_states['risk_adj_debt_usd']).fillna(np.inf)
```
Comment (lukaspetrasek) [inline src/loans/kamino.py:354]: Fillna not necessary. NAs are fine.

Comment (Chepelau) [reply src/loans/kamino.py:354]: The the sort_values method would hate it wouldn't it?

**Comment by nickgore** [comment]
lgtm :+1:

**Pull request merged**